contributing/development.md

```diff
@@ -23,6 +23,14 @@ You should now have a running server! Visit [localhost:4000](http://localhost:40
 
 When you're ready to stop your local server, type <kbd>CTRL</kbd><kbd>c</kbd> in your terminal window.
 
+### Using GitHub Codespaces
+
+As an alternative, you can simply use [GitHub Codespaces](https://github.com/features/codespaces).
+
+In a matter of minutes, you will be ready to edit, preview and test your changes directly from the comfort of your browser.
+
+![](../assets/images/codespaces-with-preview.jpg)
```

Comment:
One last one @ojacques, would you mind removing the image from here too, to keep things consistent? Once that's done, I'll get this merged! Thank you so much for helping our Codespaces users get set up 🚀

Reply:
@janiceilene - done. It became a really small PR eventually 😄. Simple is beautiful.

Reply:
Thanks so much for iterating with us @ojacques! I'll get this branch and get this merged once tests are green 🍏